editable-templates/mvp-reference-pages/table-component.html

```diff
@@ -0,0 +1,69 @@
+---
+title: Table component
+hide_breadcrumb: false
+date_modified: 2024-10-31
+layout: no-container
+---
+
+<div class="">
+	<div class="container">
+		<div class="">
+			<div class="row">
+				<div class="title col-lg-12 col-xs-12">
+					<h1 property="name" id="wb-cont" dir="ltr"> Table component
+					</h1>
+				</div>
+				<div class="table-responsive col-lg-12">
+					<table class="table table-bordered">
+						<tbody>
```

Comment:
If possible, there will be no `<tbody>` element here. Except if you took this HTML from the "live DOM" because I know that the browser are adding it automatically. But if you took that source code from "View page source" then it should not be there.